NodeManager.cpp

```diff
@@ -1318,6 +1398,66 @@ void SensorBH1750::onReceive(const MyMessage & message) {
 #endif
 
 /*
+   SensorTSL2561
+*/
+#if MODULE_TSL2561 == 1
+// contructor
+SensorTSL2561::SensorTSL2561(int child_id): Sensor(child_id,A4) {
+  setPresentation(S_LIGHT_LEVEL);
+  setType(V_LEVEL);
+  _tsl = new TLS2561(TSL2561_ADDR_FLOAT);
```

Comment:
As descripted in the example ino file from the TSL2561 library :
// The address will be different depending on whether you let
// the ADDR pin float (addr 0x39), or tie it to ground or vcc. In those cases
// use TSL2561_ADDR_LOW (0x29) or TSL2561_ADDR_HIGH (0x49) respectively
TSL2561 tsl(TSL2561_ADDR_FLOAT); 

So we have these 3 options, so let's create another setter ?